### PR TITLE
Bump `vite` to v3 and `@sveltejs/kit` to 1.0.0-next.405

### LINKED ISSUE #54: Use lite YouTube embeds
I believe someone complained about our YouTube embeds taking long to load and/or setting cookies.

Both problems might be solved by this 3rd party project https://github.com/paulirish/lite-youtube-embed which claims to be 200x faster and use youtube-nocookie.com URLs.

Anyone want to give it a try?

### LINKED ISSUE #59: Check Lighthouse test
Lighthouse test is always failing because of performance (due to the map i suppose).
Could you take a look at this test? @janosh

### PR DESCRIPTION
**Pull request by janosh**
Closes #54 and closes #59.

This PR grew a lot to include fixes for significant breaking changes in SvelteKit's file-system based router leading up to the v1.0 release (see https://github.com/sveltejs/kit/discussions/5774).

### CODE REVIEW

**Comment by janosh** [comment]
Looks like f0ef71d was enough to fix the long-standing lighthouse perf issue from slow youtube embeds on the `/airtable` page. So this PR closes #59 and #54.

File: src/routes/signup-pupil.svelte [hunk 1]
```diff
@@ -15,9 +15,8 @@
   import { signup_form_submit_handler } from '../utils/airtable'
 
   export const load: Load = async () => {
-    const chapters = (await fetchChapters()).filter(
-      (chap: Chapter) => chap.acceptsSignups
-    )
+    let chapters = await fetchChapters()
```
Comment (kleinicke) [inline src/routes/signup-pupil.svelte:18]: what does this change? Could it possibly break the signup?

**Review by kleinicke — APPROVED** [review]

**Comment by kleinicke** [comment]
I tested it for Studenten-bilden-schueler.de and the signup process works.
For students-for-students.org some some errors occur during building that can be fixed later.

File: src/routes/signup-pupil.svelte (hunk 1 above)
Comment (janosh) [reply src/routes/signup-pupil.svelte:18]: Just a refactor for readability. The code is equivalent.

**Pull request merged**